src/components/ScenarioParameters/components/ScenarioActions/ScenarioActions.spec.js

```diff
@@ -4,12 +4,12 @@
 /* eslint-disable react/prop-types */
 import React from 'react';
 import { ButtonTesting } from '../../../../../tests/MuiComponentsTesting';
-import { customRender } from '../../../../../tests/utils/renderInThemeAndStoreProviders';
+import { customRender, getByDataCy } from '../../../../../tests/utils';
 import { ScenarioActions } from './';
-import { getByDataCy } from '../../../../../tests/utils';
 import { SCENARIO_RUN_STATE } from '../../../../services/config/ApiConstants';
 import { useFormState } from 'react-hook-form';
 import { useCurrentScenarioState } from '../../../../state/hooks/ScenarioHooks';
+import { TypographyTesting } from '../../../../../tests/MuiComponentsTesting/TypographyTesting';
```

Comment:
There's another import of `ButtonTesting` from the `MuiComponentsTesting` folder, maybe we can group these imports?